app/templates/views/pricing/how-to-pay.html

```diff
@@ -23,32 +27,42 @@ <h2 class="heading-medium" id="invoices">
     </ul>
 
     <p class="govuk-body">
-      If your organisation has more than one service, you'll see a breakdown of each service on your invoice.
+      If your organisation has more than one Notify service, you’ll see a breakdown of each service on your invoice.
     </p>
 
     <p class="govuk-body">
-      If the value of an invoice is less than £250 (before <abbr title="Value Added Tax">VAT</abbr>), we’ll add it to the total for the next quarter to save time and effort.
+      If the value of an invoice is less than £250 (before <abbr title="Value Added Tax">VAT</abbr>), it will be added to the total for the next quarter to save time and effort.
     </p>
 
     <p class="govuk-body">
       You can pay by BACS, debit card, or credit card.
     </p>
 
+    <p class="govuk-body">
+      Please start your payment reference with the invoice number.
+    </p>
+
     <h2 class="heading-medium" id="purchase-orders">
       Purchase orders
     </h2>
 
     <p class="govuk-body">
-      If your organisation’s estimated spend is more than £500 per quarter (before <abbr title="Value Added Tax">VAT</abbr>), you need to send us a purchase order (PO).
+      If your organisation’s total estimated spend is more than £500 per quarter (before <abbr title="Value Added Tax">VAT</abbr>), you need to raise a purchase order (PO).
     </p>
 
     <p class="govuk-body">
-      Your organisation should raise a single <abbr title="purchase order">PO</abbr> for the estimated cost of all its services. You can update the PO anytime if your usage increases.
+      Your organisation should raise a single <abbr title="purchase order">PO</abbr> for the estimated cost of all its services. You can update the <abbr title="purchase order">PO</abbr> any time if your usage increases.
     </p>
 
+      {% if current_service and current_service.trial_mode %}    
+    <p class="govuk-body">
+      Your organisation may need to <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.billing_details') }}">add the Cabinet Office as a supplier</a> before you can raise a <abbr title="purchase order">PO</abbr>.
+    </p>
+      {% else %}
     <p class="govuk-body">
-      You may need to set up the Cabinet Office as a supplier before you can raise a <abbr title="purchase order">PO</abbr>.
+      Your organisation may need to <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.billing_details_signed_out') }}">add the Cabinet Office as a supplier</a> before you can raise a <abbr title="purchase order">PO</abbr>.
```

Comment:
@quis is this link correct?

Reply:
Use this one and it will show the right content depending on whether the user is signed in:
```suggestion
      Your organisation may need to <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.billing_details') }}">add the Cabinet Office as a supplier</a> before you can raise a <abbr title="purchase order">PO</abbr>.
```

Reply:
Nice. In that case, we don’t need the If statement anymore. :-)